### PR TITLE
Restmapper from config

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward https://github.com/giantswarm/giantswarm/issues/16406

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -8,7 +8,7 @@ require (
 	github.com/giantswarm/appcatalog v0.4.1
 	github.com/giantswarm/backoff v0.2.0
 	github.com/giantswarm/exporterkit v0.2.1
-	github.com/giantswarm/helmclient/v4 v4.3.0
+	github.com/giantswarm/helmclient/v4 v4.4.1-0.20210325152910-40b81b05f9f2
```
Comment (tomahawk28) [inline go.mod:11]: TODO: revert it back

**Comment by tomahawk28** [comment]
Tested in `ghost`

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Handover RESTMapper object to helmclient to reduce the number of REST API calls.
```
Comment (rossf7) [inline CHANGELOG.md:12]:
```suggestion
- Pass RESTMapper to helmclient to reduce the number of REST API calls.
```

Could we go with this as its a bit shorter?

**Review by rossf7 — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 2 above)
Comment (tomahawk28) [reply CHANGELOG.md:12]: Looks better, Thanks!

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Pass RESTMapper to helmclient to reduce the number of REST API calls.
```
Comment (rossf7) [inline CHANGELOG.md:12]:
```suggestion
- Pass RESTMapper to helmclient to reduce the number of REST API calls.
- Updated Helm to v3.5.3.
```

@tomahawk28 Can we add this?

So we don't need https://github.com/giantswarm/chart-operator/pull/727

Comment (tomahawk28) [reply CHANGELOG.md:12]: Sure, looks good 👍

**Pull request merged**